src/core/public/fatal_errors/fatal_errors_service.tsx

```diff
@@ -86,14 +92,18 @@ export class FatalErrorsService {
     const container = document.createElement('div');
     this.params.rootDomElement.appendChild(container);
 
+    // If error occurred before I18nService has been started we don't have any
+    // i18n context to provide.
+    const I18nContext = this.i18n ? this.i18n.Context : React.Fragment;
```

Comment:
__note:__ not really happy about that, but `fatal_errors` is a special service that can handle errors even before it's started... But at the same time I'd try avoid exposing Context on I18nService before it's started so that we start translating EUI labels only on start. Please let me know if you have a better idea.

Reply:
I think this is a fine idea, you're right about fatalErrors being special so this shouldn't be necessary anywhere else 👍